data.sql

```diff
@@ -11,3 +11,13 @@ INSERT INTO animals (name, date_of_birth, escape_attempt, weight_kg, neutered) V
 INSERT INTO animals (name, date_of_birth, escape_attempt, weight_kg, neutered) VALUES ('Pikachu', '2021-01-7', 1, 15.04, false);   
```

Comment:
- [ ] It would be better if we add a new column called species with a string type in the schema.sql as in the screenshot below

![Screenshot from 2022-08-10 14-05-30](https://user-images.githubusercontent.com/46388113/183921941-0ddd2714-6e08-4087-9181-c3f021ea8aa0.png)